frontend/src/components/BaseDashboard/components/DashboardWidgets/index.js

```diff
@@ -0,0 +1,38 @@
+import DashboardWidget from "../DashboardWidget";
+import React from "react";
+
+const DashboardWidgets = () => (
+  <>
+    <strong>Date colectate</strong>
+    <br />
+    <br />
+    <div className="columns is-desktop">
+      <div className="column is-one-third-desktop">
+        <div className="dashboard-box" style={{ padding: 0 }}>
+          <DashboardWidget
+            title={"Utilizatori red flag in ultimile 24 de ore"}
+            data={[100, 200, 300, 200]}
```

Comment:
you should put those numbers in a constant (they look random at the moment - even more so in a week from now)